integration-tests/cli/helpers/custom-objects-export.data.js

```diff
@@ -0,0 +1,59 @@
+export const createdCustomObjects = [
+  {
+    container: 'Ludus',
+    key: 'copperKey',
+    value: {
+      paymentMethod: 'Cash',
+      paymentID: '1',
+    },
+  },
+  {
+    container: 'Frobozz',
+    key: 'jadeKey',
+    value: {
+      paymentMethod: 'cc',
+      paymentID: '2',
+    },
+  },
+  {
+    container: 'Syrinx',
+    key: 'crystalKey',
+    value: {
+      paymentMethod: 'new',
+      paymentID: '3',
+    },
+  },
+]
+
+export const expectedCustomObjects = [
```

Comment:
Instead of creating two arrays for created and expected, you can create only one, then assert that what is returned from the server contains the object you created. Then this wouldn't be needed